server/src/test/scala/za/co/absa/atum/server/api/TestTransactorProvider.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api
+
+import doobie.HC
+import doobie.util.transactor.{Strategy, Transactor}
+import za.co.absa.atum.server.config.PostgresConfig
+import zio._
+import zio.interop.catz._
+
+object TestTransactorProvider {
+
+  val layer: ZLayer[Any, Config.Error, Transactor[Task]] = ZLayer {
```

Comment:
I really like that you implemented both layers like this. I would consider further distinguishing between those two by renaming this one to `layerWithoutRollback` perhaps

Reply:
(also todo for me, I want to ask about some details related to this and differences to the TransactorProvider that lives outside tests)

Reply:
Ok, I have renamed it as per your suggestion.